pepper-apis/docs/specification/src/components/responses.yml

```diff
@@ -463,9 +463,15 @@ CreateFileUploadUrlResponse:
           url:
             type: string
             description: Signed URL which needs to be used for file upload
-          guid:
+          httpMethod:
```

Comment:
Stated this in Yuri's PR, but this is encoding implementation details about Google's Bucket API into our API. Developers using this API should be familiar with the bucket APIs, and this be a boolean `resumable`, and let the consumers of the api handle it appropriately

Reply:
It does look like URL and the resumable boolean is enough for client to figure out what to do.
577f5cfd